src/Api/Translation.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE file for details.
+ */
+
+namespace FAPI\PhraseApp\Api;
+
+use FAPI\PhraseApp\Model\Translation\TranslationDeleted;
+use FAPI\PhraseApp\Model\Translation\Translation as TranslationModel;
+use GuzzleHttp\Psr7\Response;
+use Psr\Http\Message\ResponseInterface;
+use FAPI\PhraseApp\Exception;
+
+/**
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
+ */
+class Translation extends HttpApi
+{
+    /**
+     * Get a translation.
+     *
+     * @param string $projectKey
+     * @param string $id
+     * @param string $localeId
+     *
+     * @return TranslationModel|ResponseInterface
+     *
+     * @throws Exception
+     */
+    public function get(string $projectKey, string $domain, string $id, string $localeId)
```

Comment:
You do not need domain. I think we should be more general in the client. Maybe use $query as parameter?